ui/dialogs/albuminfodialog.go

```diff
@@ -0,0 +1,118 @@
+package dialogs
+
+import (
+	"fmt"
+	"image"
+	"net/url"
+	"strings"
+
+	"github.com/dweymouth/supersonic/backend/mediaprovider"
+	"github.com/dweymouth/supersonic/ui/layouts"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/canvas"
+	"fyne.io/fyne/v2/container"
+
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+const musicBrainzReleaseUrl = "https://musicbrainz.org/release"
+
+type AlbumInfoDialog struct {
+	widget.BaseWidget
+
+	OnDismiss func()
+
+	content fyne.CanvasObject
+}
+
+func NewAlbumInfoDialog(albumInfo *mediaprovider.AlbumInfo, albumName string, albumCover image.Image) *AlbumInfoDialog {
+	a := &AlbumInfoDialog{}
+	a.ExtendBaseWidget(a)
+
+	a.content = container.NewVBox(
+		a.buildMainContainer(albumInfo, albumName, albumCover),
+		widget.NewSeparator(),
+		container.NewHBox(
+			layout.NewSpacer(),
+			widget.NewButton("Close", func() {
+				if a.OnDismiss != nil {
+					a.OnDismiss()
+				}
+			}),
+		),
+	)
+	a.content.Resize(a.MinSize())
```

Comment:
You don't need this line - the Fyne layout system will automatically resize the content when it's being shown